data.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO animals (id,name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES (1, 'Agumon', '2020-02-03', 0, TRUE, 10.23);
```

Comment:
The project looks awesome but it misses a professional README file. You may follow [this](https://github.com/microverseinc/readme-template) template from Microverse and customize it according to project needs.

Reply:
Sure. Thank you